src/qml/nodemodel.cpp

```diff
@@ -27,14 +28,60 @@ void NodeModel::setBlockTipHeight(int new_height)
     }
 }
 
+void NodeModel::setRemainingSyncTime(double new_progress)
```

Comment:
This function is pretty confusing in general, we'll heavily change this in follow-ups

Reply:
should probably do an early test on new_progress to make sure it's a valid value